packages/react-ai/src/components/AIConversation/views/Controls/FieldControl.tsx

```diff
@@ -168,7 +168,7 @@ export const FieldControl: FieldControl = () => {
         const fileContent: ConversationMessageContent = {
           image: {
             format: getImageTypeFromMimeType(file.type),
-            source: { bytes: Uint8Array.from(Buffer.from(buffer)) },
+            source: { bytes: new Uint8Array(buffer) },
```

Comment:
Yep have it as a todo, the test wasn't set up yet, but it wouldn't change the behavior here since except for catchign Buffer not being polyfilled maybe?